doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -180,80 +178,65 @@ Requirements for software and settings:
+```
+curl https://artifactory.niis.org/api/gpg/key/public | sudo apt-key add -
+```
 
-  ```
-  adapter=postgresql
-  encoding=utf8
-  username=<database user>
-  password=<database password>
-  database=<database name>
-  reconnect=true
-  host=<database host>
-  port=<database port>
-  schema=<database schema>
-  ```
+Add X-Road package repository (**reference data: 1.1**)
+```
+sudo apt-add-repository -y "deb https://artifactory.niis.org/xroad-release-deb $(lsb_release -sc)-current main"
+```
 
-### 2.6 Remote Database Installation
+**This is an optional step.** Add a package repository for an alternative Java distribution. According to [the Ubuntu blog](https://ubuntu.com/blog/announcing-openjdk-11-packages-in-ubuntu-18-04-lts), Ubuntu OpenJDK 8 security updates end in April 2021. [AdoptOpenJDK](https://adoptopenjdk.net/) is an open-source Java 8 distribution that is [supported until May, 2026](https://adoptopenjdk.net/support.html#roadmap).
 
-**This is an optional step.** In case you want to install central server with remote database, perform the steps in this chapter. Otherwise, skip it and continue from the next chapter.
+```
+curl https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -
+sudo apt-add-repository -y "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb $(lsb_release -sc) main"
+```
 
-When using a remote database server with the central server, you should verify that the version of the local PostgreSQL client matches the version of the remote PostgreSQL server.
+### 2.6 Remote Database Setup (optional)
 
-To use a remote database server instead of the default locally installed one, you need to pre-create a configuration file containing the database administrator master password. If storing the database administrator password on the central server is not possible due to security risk or other problem, the alternative is to create the database structure manually as described in [Annex D Create Database Structure Manually](#annex-d-create-database-structure-manually). Otherwise, creating the configuration file can be done by performing the following steps:
+*This is an optional step.* 
 
-  ```
-  sudo touch /etc/xroad.properties
-  sudo chown root:root /etc/xroad.properties
-  sudo chmod 600 /etc/xroad.properties
-  ```
+Optionally, the central server can use a remote database server. To avoid installing the default local PostgreSQL server during the installation, first install the `xroad-database-remote` -package.
+```
+sudo apt install xroad-database-remote
+```
 
-Edit `/etc/xroad.properties` contents. See the example below. Replace the parameter values with your own.
+For backup and restore to work correctly, it is important to verify that the local PostgreSQL client has the same or later major version than the remote database server and, if necessary, install a different version of the `postgresql-client` package (see https://www.postgresql.org/download/linux/ubuntu/)
```

Comment:
> For backup and restore to work correctly

It might be good to clarify that this refers to the Security Server application level back up and restore. For example:

> For the security server application level backup and restore to work correctly